source/ControlBackend.h

```diff
@@ -83,6 +81,15 @@ class ControlBackend final : public ControlBackendInterface
         return ApplicationState::RUNNING == m_rialtoClientState;
     }
 
+    void registerGstLogHandler() override
```

Comment:
Maybe we could register the handler in Control's constructor? The sooner we register it, the less logs we loose. Loosing logs from IPC connection could be brutal :(